sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -24,6 +24,7 @@ def plotter():
 
 def test_basic(plotter):
     assert isinstance(plotter.plotter, pv.Plotter)
+    plotter.show()
```

Comment:
Will this display a figure?

Reply:
PYVISTA_OFF_SCREEN = True would prevent this from happening right?

Reply:
So the reason for adding the show was to make sure it works?

Reply:
yes, I think it returns a cpos so I can assert that, let me check

Reply:
Yeah it returns nothing, the docs say otherwise but I think they've changed that.
Yeah `show` was added to make sure it works with the ` PYVISTA_OFF_SCREEN = True`